docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+    
+    dalalserver:
+        container_name: dalalstreet-server
+        image: dalalstreet_server
+        build: .
+        restart: always
+        volumes: 
+            - ./docker/logs/:/dalal-street-server/logs/
+        ports:
+            - "${SERVER_PORT}:8000"
+        depends_on:
+            - db
+        env_file:
+            - .env
+        environment:
+            MYSQL_DATABASE: ${DB_NAME}
+            DALAL_ENV: Docker
+            MYSQL_ROOT_PASSWORD: ${DB_PASS}
```

Comment:
Could you put everything in `.env` file itself? `DB_NAME` and `DB_PASS` are not being used anywhere. Directly put `MSQL_ROOT_DATABASE`, `MSQL_ROOT_PASSWORD` and `DALAL_ENV` in `.env` file and remove the environment section in compose file.